shared/templates/sysctl/tests/one_sysctl_conf_one_sysctl_d.pass.sh

```diff
@@ -4,7 +4,11 @@
 {{% endif %}}
 
 # Clean sysctl config directories
+{{% if "ubuntu" in product %}}
+rm -rf /usr/lib/sysctl.d/* /run/sysctl.d/* /etc/sysctl.d/* /etc/ufw/sysctl.conf
```

Comment:
Maybe it'd a good idea to have this either as a product property or as a template parameter. At this point it all looks like littering in the template.

Reply:
This is a note for the change in master, not for back-porting PR.